### ISSUE TITLE
NTR: Mycobacterium tuberculosis in CSF PCR positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8058]
New term request Mycobacterium tuberculosis in CSF PCR positive (samant_101122005636): Cerebrospinal fluid; polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000542
- label: Positive CSF mycobacterium tuberculosis nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of Mycobacterium tuberculosis nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8058
- synonyms:
